src/components/molecules/MatchMeToBenefits.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ActionButton } from "../atoms/ActionButton";
+import PropTypes from "prop-types";
+
+/**
+ * This component shows match me to benefits button and questions when authenticated and clicked
+ */
+export function MatchMeToBenefits(props) {
```

Comment:
why create a separate button component as opposed to just using the ActionButton on the homepage

Reply:
Originally I planned on having the button/question together but that has changed. I'll move it back